### DISCUSSION TITLE
Changing pywinstyles by ctkSwitch to Deafault color theme

### DISCUSSION THREAD

**Discussion by CheckAvaliable** [Akascape/py-window-styles#3] [Q&A] [answered]
I have a ctkSwitchbox with which I want to change the standard ctk theme to aero from pywintyles. The problem occurs after applying aero - I just can't get it back. And yes I am using windows 10 and I know that some pywinstyles themes do not work in this OS. Therein lies the main point - optimised, dark and light themes don't work in 10

**Comment by Akascape** [comment, marked as answer]
@CheckAvaliable 

> I have a ctkSwitchbox with which I want to change the standard ctk theme to aero from pywintyles. The problem occurs after applying aero - I just can't get it back.

Pywinstyles works only with the root/toplevel windows. So use it with the main window and not the widgets. If you want to get the main window of any widget, use `widget.winfo_toplevel()` and then pass it to pywinstyles.

> optimised, dark and light themes don't work in 10

dark theme will work after the new update, just update pywinstyles. `pip install pywinstyles --upgrade`

**Comment by CheckAvaliable** [comment]
Okay, thanks. I will now try to update and use the dark theme.
